### DISCUSSION TITLE
MOD exchanges !!

### DISCUSSION THREAD

**Discussion by Romanitho** [Romanitho/Winget-AutoUpdate#252] [MODs]
Share your mods with us 😇 !!

Here is the place where you can discuss, share, pick,... different mods for your Winget programs.

**Comment by Romanitho** [comment]
@KnifMelti We can share our best MODs with others 😁  
Maybe we can think about a naming convention for titles.

**Reply by KnifMelti** [reply]
Title name could be the mod file name.

**Reply by Romanitho** [reply]
perfect :)
